docs/bond-rest-api-spec.yaml

```diff
@@ -0,0 +1,1303 @@
+                authorizations:
+                  - public_key: 02178c1bcdb25407394348f1ff5273adae287d8ea328184546837957e71c7de57a
+                    role: MARKETMAKER
+      responses:
+        '200':
+          $ref: '#/components/responses/SuccessResponse'
+        '400':
+          $ref: '#/components/responses/BadRequestError'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    get:
+      description: Returns all organizations
+      parameters:
+        - $ref: '#/components/parameters/head'
+        - $ref: '#/components/parameters/start'
+        - $ref: '#/components/parameters/limit'
+      responses:
+        '200':
+          description: Success response with array of organizations
+          content:
+            'application/json':
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/OrganizationModel'
+                  head:
+                    $ref: '#/components/schemas/HeadModel'
+                  link:
+                    $ref: '#/components/schemas/LinkModel'
+              example:
+                data:
+                  - organization_id: AVCD
+                    name: Bank of Terra Incognita
+                    organization_type: Pricing Source
+                    industry: banking
+                    holdings:
+                      - asset_type: Currency
+                        asset_id: USD
+                        amount: 100000000000
+                    authorizations:
+                      - public_key: 02178c1bcdb25407394348f1ff5273adae287d8ea328184546837957e71c7de57a
+                        role: MARKETMAKER
+                    receipts:
+                      - bond_id: US00206RDA77
+                        payment_type: COUPON
+                        coupon_date: 1333170000
+                        amount: 15000000
+                        timestamp: 1333170042
+        '400':
+          $ref: '#/components/responses/BadRequestError'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /organizations/{organization_id}:
+    parameters:
+      - in: path
+        name: organization_id
+        schema:
+          type: string
+        required: true
+        description: ISIN of an organization
+        example: AVCD
+    post:
+      description: Submits a transaction to update an organization
+      security:
+        - bearerAuth: []
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              type: object
+              properties:
+                name:
+                  description: Updated name of the organization
+                  type: string
+                organization_type:
+                  $ref: '#/components/schemas/OrganizationTypeEnum'
```

Comment:
As we decided yesterday the organization type should not be changed. So, this should be removed.